dls-controls/malcolmjs
# Feature/fix duplications #198

## Description

Removed duplications (mainly in test code).
Fixed some code smells identified by SonarQube.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Review updates to https://sonarcloud.io/dashboard?id=malcolmjs

## Agile board tracking

connect to #198 
closes #198
